package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "nuclear-js-react-addons",
+  "version": "1.0.0",
```

Comment:
lets keep this `0.1.0` as to be more flexible to API changes if we need to make them.